src/Bridges/vaftosafbridge.jl

```diff
@@ -0,0 +1,148 @@
+const VectorToScalarBridgeSets = Union{MOI.Zeros,
+                                       MOI.Nonpositives,
+                                       MOI.Nonnegatives}
+get_scalar_set(T, ::Type{MOI.Zeros}) = MOI.EqualTo{T}
+get_scalar_set(T, ::Type{MOI.Nonpositives}) = MOI.LessThan{T}
+get_scalar_set(T, ::Type{MOI.Nonnegatives}) = MOI.GreaterThan{T}
+
+get_vector_set(::Type{MOI.EqualTo{T}}) where T = MOI.Zeros
+get_vector_set(::Type{MOI.LessThan{T}}) where T = MOI.Nonpositives
+get_vector_set(::Type{MOI.GreaterThan{T}}) where T = MOI.Nonnegatives
+
+"""
+    VectorToScalarBridge{T, S}
+
+The `VectorToScalarBridge` splits an `VectorAffineFunction` into a series of
+`ScalarAffineFunction`s.
+
+Supported sets are:
+ - `MOI.Zeros` =>`MOI.EqualTo{T}`
+ - `MOI.Nonnegatives` =>`MOI.GreaterThan{T}`
+ - `MOI.Nonpositives` =>`MOI.LessThan{T}`
+"""
+struct VectorToScalarBridge{T, S} <: AbstractBridge
+    constraints::Vector{CI{MOI.ScalarAffineFunction{T}, S}}
+end
+function VectorToScalarBridge{T, ScalarSet}(model,
+                                            func::MOI.VectorAffineFunction{T},
+                                            set::VectorSet) where
+                                            {T, ScalarSet, VectorSet}
+    @assert ScalarSet == get_scalar_set(T, VectorSet)
+    sets = [ScalarSet(-constant) for constant in func.constants]
+    @assert MOI.dimension(set) == length(sets)
+    functions = [MOI.ScalarAffineFunction{T}(MOI.ScalarAffineTerm{T}[], zero(T))
+        for i in 1:MOI.dimension(set)]
```

Comment:
Cosmetic/Subjective: I would indent to start `for` just past the `[`

Reply:
indenting is where coder can express their artistic inner-self :D 
More seriously, I was confused when reading the line as I thought it was a block (e.g. like `for`, `begin`, `if`, ...) instead of a line-break.
I usually do 4-spaces for blocks and for line-breaks I align with parentheses, brackets, ...